internal/guest/runtime/hcsv2/spec.go

```diff
@@ -202,25 +191,13 @@ func getGroup(spec *oci.Spec, filter func(user.Group) bool) (user.Group, error)
 func applyAnnotationsToSpec(ctx context.Context, spec *oci.Spec) error {
 	// Check if we need to override container's /dev/shm
 	if val, ok := spec.Annotations[annotations.LCOWDevShmSizeInKb]; ok {
-		sz, err := strconv.ParseInt(val, 10, 64)
+		mt, err := devShmMountWithSize(val)
```

Comment:
Why did we refactor this?

Reply:
`/dev/shm` mount is part of default mounts that CRI/opengcs adds. Security policy enforces exact values for mount options, so it means we need a way to generate the mount with proper size, so I moved the mount spec creation into its own function. I haven't added the logic, which adds the actual mount policy, when different shared memory size is requested though. I'll do it in a separate PR, since we don't have a use case for it at the moment.

Reply:
That makes sense. Seems kind of weird to me though that we explicitly use the "/dev/shm" string to try to remove the mount below even though we've refactored out the hardcoded string here when setting up.

Reply:
Technically, we've refactored just the mount spec object creation, similar to how we'd do something like `NewThing`, leaving the actual spec update here.

Reply:
Right, I just meant it feels wrong to remove one hardcoded string reference and keep the other

Reply:
gocha gocha. added a const.